atomic_reactor/plugins/pre_resolve_remote_source.py

```diff
@@ -70,7 +75,10 @@ def run(self):
             self.log.info('Aborting plugin execution: missing Cachito configuration')
             return
 
-        if self.workflow.source.config.remote_sources:
+        if (
```

Comment:
can you change formatting here? this looks odd

if (...........................  looks nicer